YAFCui/ImGui/ScrollArea.cs

```diff
@@ -13,6 +13,7 @@ public abstract class Scrollable : IKeyboardFocus {
         private float height;
         private ImGui gui;
         public const float ScrollbarSize = 1f;
+        private const float BottomPaddingInPixels = 100f;
```

Comment:
Would be nice to add a comment (to new constants, functions, etc.) telling what this is and why it is here. In this case something like

"Padding to add at the bottom of the scroll area to be able to scroll past the last item"

or something

Reply:
Done in b4b39bd